cg/meta/transfer/flowcell.py

```diff
@@ -110,8 +110,8 @@ def _parse_flow_cell_samples(
                     sequencing_files=cgstats_sample.fastqs,
                     tag_name=SequencingFileTag.FASTQ,
                 )
-
-            status_db_sample.reads = cgstats_sample.reads
+            # commenting out for now - just for traceability. This code will be removed later.
```

Comment:
Oh no zombie code! 🧟

Reply:
I guess this entire module is supposed to be nuked so I'm not sure if we should touch it

Reply:
yes you are right we have the PRs that keep track of code, I should remove it.